src/netappfiles-preview/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.3.2"
+VERSION = "0.4.2"
```

Comment:
Please follow this [doc](https://microsoft.sharepoint.com/:w:/t/AzureCLITools237/EWG5EuACKmtNum6A3pyLakMBSaQnkjomC9XPz9ogNPS69A?e=CWDIxF) for the versioning of Azure CLI Extension
```suggestion
VERSION = "1.0.0b1"
```